src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -113,6 +113,7 @@ private static void ReturnBuffer(CommandData[] cmds)
             using (var parser = new UnmanagedJsonParser(ctx, state, "bulk_docs"))
             using (var modifier = new BlittableMetadataModifier(ctx))
             {
+                modifier.OperateOnTypes |= DatabaseItemType.Attachments;
```

Comment:
Why we have this here? I would prefer to add this via ctor and with appropriate comment